File: prime.py
# Program to find prime numbers up to a given number

num = int(input("Enter a number: "))

print(f"Prime numbers up to {num} are:")

for n in range(2, num + 1):
    is_prime = True
    
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            is_prime = False
            break
    
    if is_prime:
        print(n, end=" ")

File: prime.php
<?php
function isPrime($num)
{
    if ($num <= 1)
        return "Not Prime";

    for ($i = 2; $i <= $num / 2; $i++) {
        if ($num % $i == 0) {
            return "Not Prime";
        }
    }
    return "Prime";
}

echo "7 is " . isPrime(7);
?>